docs/source/tutorial/calc_tutorial.rst

```diff
@@ -269,52 +276,28 @@ in place of calling :func:`.detector_grid`.
 Scattering Theories in HoloPy
 -----------------------------
 
-HoloPy contains a number of scattering theories to model the scattering
-from different kinds of scatterers. By default, scattering from single
-spheres is calculated using Mie theory, which is the exact solution
-to Maxwell's equations for the scattered field from a spherical
-particle, originally derived by Gustav Mie and (independently) by
-Ludvig Lorenz in the early 1900s.
-
-A scatterer composed of multiple spheres can exhibit multiple scattering
-and coupling of the near-fields of neighbouring particles. Mie theory doesn't include
-these effects, so :class:`.Spheres` objects are by default calculated using the
-SCSMFO package from `Daniel Mackowski <http://www.eng.auburn.edu/~dmckwski/>`_.
-This calculation uses T-matrix methods to give the exact solution to Maxwell's equation
-for the scattering from an arbitrary arrangement of non-overlapping spheres.
-
-Sometimes you might want to calculate scattering from multiple spheres
-using Mie theory if you are worried about computation time,
-if your spheres are widely separated (such that optical coupling between
-the spheres is negligible),
-or if you are
-using multi-layered spheres (HoloPy's implementation of the multisphere theory
-can't currently handle coated spheres). You can specify Mie theory manually when
-calling the :func:`.calc_holo` function:
+HoloPy contains a number of scattering theories to model the scattering from
+different kinds of scatterers. You can specifiy a scattering theory by setting the `theory` keyword to a :class:`.ScatteringTheory` object, rather than setting the theory to `'auto'`. For instance, to force `holopy` to calculate the hologram of a sphere using Mie theory (the theory which exactly describes scattering from a spherical particle), we set the `theory` keyword to an instance of the :class:`.Mie` class:
 
-..  testcode::
-
-    from holopy.scattering import Mie
-    holo = calc_holo(exp_img, collection, theory = Mie)
 
 ..  testcode::
-    :hide:
-
-    print(holo[0,0,0].values)
-
-..  testoutput::
-    :hide:
-
-    1.04802354...
 
-Similarly, HoloPy calculates scattering from cylindrical or spheroidal particles by using T-matrix code from `Michael Mishchenko <https://www.giss.nasa.gov/staff/mmishchenko/t_matrix.html>`_, but these scatterer types are not compatible with Mie theory.
+    theory = Mie()
```

Comment:
Now we need a Mie import. I know it's a bit annoying to have imports scattered everywhere, but it's an effort to keep the docs modular so you can just look at the sections you are interested in.

Reply:
Should be fixed by e8a2dfde